frontend/packages/topology/src/components/graph-view/components/componentUtils.ts

```diff
@@ -226,7 +227,9 @@ const graphDropTargetSpec: DropTargetSpec<
   collect: (monitor) => {
     const operation = monitor.getOperation() as EditableDragOperationType;
     const dragEditInProgress =
-      monitor.isDragging() && (operation?.type === CREATE_CONNECTOR_OPERATION || operation?.edit);
+      monitor.isDragging() &&
+      (operation?.type === CREATE_CONNECTOR_OPERATION || operation?.edit) &&
+      operation?.type !== CREATE_EV_SRC_KAFKA_CONNECTOR_OPERATION;
```

Comment:
@jeff-phillips-18  Does this looks fine, we added it here as we needed to hide + actin on the drag of connector cc @vikram-raj

Reply:
I would have expected a `canDrop` check or something here. No point setting `dragEditInProgress` if you can't drop on the graph. Should be a better way than checking specifically for that type.